modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/internal/portlet/ContactsCenterPortlet.java

```diff
@@ -432,7 +432,12 @@ public void requestSocialRelation(
 
 			JSONObject extraDataJSONObject = JSONFactoryUtil.createJSONObject();
 
-			String portletId = portal.getPortletId(actionRequest);
+			String portletId = PortletIdCodec.decodePortletName(
+					portal.getPortletId(actionRequest));
+
+				if (portletId.equals(ContactsPortletKeys.PROFILE)) {
+					portletId = ContactsPortletKeys.CONTACTS_CENTER;
+				}
 
 			extraDataJSONObject.put(
 				"portletId", PortletIdCodec.decodePortletName(portletId));
```

Comment:
This looks left over.  We can just pass in `portletId` instead of decoding it again.